app/views/books/show.html.erb

```diff
@@ -14,15 +14,15 @@
     <% unless @book.reviews.empty? %>
       <section class="top-reviews">
         <p>Top Reviews</p>
-        <% @book.grab_reviews('top', 3).each do |review| %>
-          <p>Title: <%= review.title %>, Rating: <%= review.rating %>, User: <%= link_to review.username %></p>
+        <%= @book.grab_reviews('top', 3).each do |review| %>
+          <p>Title: <%= review.title %>, Rating: <%= review.rating %>, User: <%= link_to review.username, user_index_path(review.username) %></p>
         <% end %>
       </section>
 
       <section class="bottom-reviews">
-        <p>Bottom Reviews</p>
-        <% @book.grab_reviews('bottom', 3).each do |review| %>
-          <p>Title: <%= review.title %>, Rating: <%= review.rating %>, User: <%= link_to review.username %></p>
+        <p>Bottome Reviews</p>
```

Comment:
Can you update lines 23-24? I corrected these typos in a previous PR but looks like they got missed here.
L23 - Bottom
L24 - Just the equal sign on the erb tag

Reply:
Took care of the typo and removed the errant echo tag.